dags/common/ingestion.py

```
@@ -0,0 +1,149 @@
+    """
+    api_key = os.getenv("OPENWEATHER_API_KEY")
+    if not api_key:
+        logger.error("Environment variable 'OPENWEATHER_API_KEY' is missing")
+        raise ValueError("OPENWEATHER_API_KEY is not exist")
+
+    api_url = (
+        f"https://api.openweathermap.org/data/2.5/weather?"
+        f"lat={lat}&lon={lon}&appid={api_key}&units=metric"
+    )
+    
+    response = requests.get(api_url)
```

Ruff (0.14.10)

Probable use of `requests` call without timeout

(S113)